packages/matchbox/src/components/CheckboxCard/CheckboxCard.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { CheckBox, CheckBoxOutlineBlank } from '@sparkpost/matchbox-icons';
+import { Box } from '../Box';
+import Group from './Group';
+import { StyledLabel, StyledInput, StyledUnchecked, StyledChecked, StyledHeader } from './styles';
+
+export type CheckboxCardProps = {
+  'data-id'?: string;
+  'data-track'?: string;
+  label?: React.ReactNode;
+  size?: 'small' | 'default';
+} & Omit<React.ComponentPropsWithoutRef<'input'>, 'type' | 'size'>;
+
+const CheckboxCard = React.forwardRef<HTMLInputElement, CheckboxCardProps>(function CheckboxCard(
+  props,
+  userRef,
+) {
+  const {
+    id,
+    checked,
+    children,
+    defaultChecked,
+    disabled,
+    label,
+    name,
+    onChange,
+    onFocus,
+    onBlur,
+    size = 'default',
+    value,
+    'data-id': dataId,
+    'data-track': dataTrack,
+    ...rest
+  } = props;
+
+  const isSmall = size === 'small';
+
+  return (
+    <Box data-id="radio-card">
+      <StyledInput
+        checked={checked}
+        data-id={dataId}
+        data-track={dataTrack}
+        disabled={disabled}
+        defaultChecked={defaultChecked}
+        id={id}
+        name={name}
+        onChange={onChange}
+        onFocus={onFocus}
+        onBlur={onBlur}
+        ref={userRef}
+        type="checkbox"
+        value={value}
+        {...rest}
+      />
+      <StyledLabel as="label" htmlFor={id} $size={size}>
+        <Box display="flex">
+          <Box flex="0 0 auto" width={isSmall ? '200' : '400'}>
+            <Box position="absolute" top={isSmall ? '100' : '500'} left={isSmall ? '200' : '500'}>
+              <StyledUnchecked size="1rem" as={CheckBoxOutlineBlank} />
```

Comment:
The sizing and spacing here is wrong. The checkbox on the large card is currently 12px x 12px and should be 16x16, and the spacing to the right of the checkbox should be 12px but it looks a little off

Reply:
Hmm looks like the react icons have 4px padding built in